src/main/scala/swiss/dasch/Main.scala

```diff
@@ -0,0 +1,35 @@
+package swiss.dasch
+
+import swiss.dasch.api.*
+import swiss.dasch.api.healthcheck.*
+import swiss.dasch.config.Configuration
+import swiss.dasch.config.Configuration.ApiConfig
+import swiss.dasch.domain.AssetServiceLive
+import zio.*
+import zio.config.*
+import zio.http.Server
+import zio.logging.backend.SLF4J
+
+object Main extends ZIOAppDefault {
+
+  override val bootstrap: ULayer[Unit] = Runtime.removeDefaultLoggers >>> SLF4J.slf4j
+
+  private val serverLayer =
+    ZLayer
+      .service[ApiConfig]
+      .flatMap { cfg =>
+        Server.defaultWith(_.binding(cfg.get.host, cfg.get.port))
+      }
+      .orDie
+
+  val routes = HealthCheckRoutes.app
+
+  private val program = Server.serve(routes)
```

Comment:
why `program` not `server` for instance?

Reply:
just because.